documentation/library_guide/extension_api.rst

```diff
@@ -255,17 +256,20 @@ These algorithms are declared in ``oneapi::dpl::experimental::ranges`` namespace
 In order to make these algorithm available the ``<oneapi/dpl/ranges>`` header should be included.
 Use of the range-based API requires C++17 and the C++ standard libraries coming with GCC 8.1 (or higher) or Clang 7 (or higher).
 
-The following viewable ranges are declared in ``oneapi::dpl::experimental::ranges`` namespace. Only those are allowed to use as ranges
+The following views are declared in ``oneapi::dpl::experimental::ranges`` namespace. Only those are allowed to use as ranges
 for range-based algorithms.
 
 * ``iota_view``: A range factory - generates a sequence of N elements which starts from an initial value and ends by final N-1.
+* ``generate``:  A range factory - generates a sequence of N elements each is produced by a given functional genrator.
+* ``fill ``: A range factory - generates a sequence of N elements each is equal a given value.
```

Comment:
Please update (updates in bold - no spaces between "elements" and the comma, it wouldn't let me bold the comma if it was touching the word "elements"):
``generate``:  A range factory - generates a sequence of N elements **, where** each is produced by a given functional **generator**.
``fill ``: A range factory - generates a sequence of N elements **, where** each is equal a given value.